docs/install-overview.md

```diff
@@ -6,6 +6,16 @@ In version 0.7.0 and later, you can run Ark in any namespace, which requires add
 
 In version 0.9.0 and later, you can use Ark's integration with restic, which requires additional setup. See [restic instructions][20].
 
+## Customize configuration
+
+Whether you run Ark on a cloud provider or on-premises, you can specify the name of the default volume snapshot when you set up the server:
+
+```bash
+kubectl apply -f examples/common/00-prereqs.yaml --default-volume-snapshot-locations <location-1>,<location-2>,<location-3>,...
```

Comment:
The command here is `ark server`, not `kubectl apply`.

So `ark server --default-volume-snapshot-locations gcp:gcp-default`, which is an example from testing on my laptop.